test/functional/ftests.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
```

Comment:
use llvm-lit and llvm FileCheck for functional tests, like we did for EK. it's easier (the command, the input, and the expected output all in one file, for each test)